packages/office-ui-fabric-react/src/components/ColorPicker/examples/ColorPicker.Basic.Example.tsx

```diff
@@ -41,6 +41,10 @@ export class ColorPickerBasicExample extends React.Component<{}, IBasicColorPick
           styles={colorPickerStyles}
           // The ColorPicker provides default English strings.
           // If your app is localized, you MUST provide the `strings` prop with localized strings.
+          strings={{
+            alphaAriaLabel: 'Use left and right arrow keys to change alpha value. Hold shift for a larger jump',
```

Comment:
I'm not convinced by the argument in #11415 that we need to provide instructions--the screen reader should announce that the control is a slider, which is a standard control that the user should already know how to interact with. (I only included instructions by default for the color rectangle because it's non-standard.) 

Or if we do want to provide an example of overrides it should be something like "Alpha slider. Use left and right arrow keys to change value. Hold shift for a larger jump." (lead with the important part)

Reply:
The spec says arrow left/right for increment, and pageup/down for ~10% jumps. Since we deviate a bit from the spec, it's probably good to point out. I agree that leading with the slider name is a good idea. I'll check and make sure it's not redundant. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role

Reply:
@ecraig12345 updated the string to have slider name first, with instructions after.